rpcs3/rpcs3qt/cheat_manager.cpp

```diff
@@ -347,16 +347,16 @@ std::vector<u32> cheat_engine::search(const T value, const std::vector<u32>& to_
 	}
 	else
 	{
-		// Looks through mapped memory
-		for (u32 page_ind = (0x10000 / 4096); page_ind < (0xF0000000 / 4096); page_ind++)
+		// Looks through mapped memory (removed division by 4096 for speedup reasons)
```

Comment:
Avoid commenting things in code where you just want to mention them for the pull request's reviewer.
Just mention them in commit/pull request description.

Reply:
Ok, understand